lib/integration/framework/Test/Integration/Framework/DSL.hs

```diff
@@ -2219,6 +2222,12 @@ fixturePassphrase :: Text
 fixturePassphrase =
     "cardano-wallet"
 
+fixturePrimitivePassphrase :: forall a
+    . (W.PassphraseMaxLength a, W.PassphraseMinLength a) => Passphrase a
+fixturePrimitivePassphrase = let
+      (Right pp) = fromText @(Passphrase a) fixturePassphrase
+      in pp
+
```

Comment:
```suggestion
fixturePrimitivePassphrase :: Passphrase "user"
fixturePrimitivePassphrase =
    let (Right pp) = fromText fixturePassphrase
    in pp
```

Could you use the concrete type here?

The polymorphic form is too much magic — for example, there is a potentially non-trivial conversion between `Passphrase "user"` and `Passphrase "encryption"`.

From the client perspective, only `Passphrase "user"` is relevant.